CountItems/CountManager.js

```diff
@@ -61,6 +70,25 @@ class CountManager {
                 }
             });
         }
+        if (results.dataMetrics
+            && results.dataMetrics.bucket
+            && results.dataMetrics.location
+            && results.dataMetrics.account) {
```

Comment:
This will avoid updating the store dataMetrics if either of these 3 fields are not in the object, is this something expected? (-> or if I put in differently, can `dataMtrics` not have these three fields?)

Reply:
yeah it's kind of expected, because there won't be a case that there are metrics in buckets but not in location nor account, all objects must be linked to a bucket, an account and one or multiple locations. so if there is only bucket metrics, we assume the dataMetrics is broken